Valorant

Derval-Filho opened the pull request:
**When submitting this pull request, I confirm the following (please check the boxes):**

- [x ] I have read and understood the [Hydra Themes Docs](https://docs.hydralauncher.gg/themes.html).
- [x ] I have checked that the folder of my theme contains my friend code in the format "My Awesome Theme-_my_friend_code_".
- [ x] I have checked that the screeshot file has one of the following extensions: png, webp, jpg, jpeg, avif, heic, heif.
- [ x] I have checked that my css file has the extension ".css".
- [ x] I am aware that the validation pipeline needs to pass with success for my theme to be approved.

Good example of how files should be in the PR:

![Example of files in a good PR](https://raw.githubusercontent.com/hydralauncher/hydra-themes/f61313ffdf4f1d5cfd2149583a02eb4da4e6c530/docs/pr-example.png)

Derval-Filho committed:
dededonoobverso

Derval-Filho committed:
Update theme.css

Derval-Filho committed:
dededonoobverso

zamitto commented:
Sua PR está com conflitos, consegue verificar?

Derval-Filho commented:
> Seu PR está com conflitos, consegue verificar?

I'm new to GitHub and I don't know much, hence so many errors. Sorry, and I'm also still learning to program.

chubbygrannychaser commented:
@Derval-Filho your PR currently has failing CI checks (SonarCloud Code Analysis). Could you take a look and push a fix? Thanks!

<!-- hermes-ci-reminder head:e18c09c96e4c21a51a8a070c2c4811de5a7c4bfe -->

Derval-Filho commented:
eu só nao me lembro como faz  para colocar aqui

Derval-Filho committed:
Update theme.css

your PR currently has failing CI checks (SonarCloud Code Analysis). Could you take a look and push a fix?

Derval-Filho commented:
feito

chubbygrannychaser commented:
@Derval-Filho checks are still failing for head `b0a8473` (the required `validate-themes` check is still not passing/present). Please take a look and push a fix when you can.

<!-- hermes-ci-reminder head:b0a8473ac1ba286daabc675c0fcdbc8b03a94cc8 -->

Derval-Filho commented:
How should it be? This was the first time I tried something like this. I
think I know where it's going wrong?

Em qua., 8 de abr. de 2026 às 03:52, Chubby Granny Chaser <
***@***.***> escreveu:

> *chubbygrannychaser* left a comment (hydralauncher/hydra-themes#591)
> <https://github.com/hydralauncher/hydra-themes/pull/591#issuecomment-4204370763>
>
> @Derval-Filho <https://github.com/Derval-Filho> checks are still failing
> for head b0a8473 (the required \ check is still not passing/present).
> Please take a look and push a fix when you can.\n\n
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/hydralauncher/hydra-themes/pull/591#issuecomment-4204370763>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A3NSZ3UGNVJGWCUJ4HO2Z5D4UXZJ3AVCNFSM6AAAAACLD4A6SWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHM2DEMBUGM3TANZWGM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
ATT: Derval De Araujo Filho
***@***.***
***@***.***
61999017033

Derval-Filho committed:
Update theme.css

adicionei o cabeçalho

chubbygrannychaser commented:
@Derval-Filho quick heads-up: for head `f562312`, the PR still isn't ready to merge because the required `validate-themes` check isn't passing/present. Please take a look and push a fix when you can.

<!-- hermes-ci-reminder head:f56231275cd10e79988ac1e26fd312a8c3abef0a -->

chubbygrannychaser commented:
Hi @Derval-Filho — checks are still not green for head `f562312` (the required `validate-themes` check is not passing/present). Please take a look and push a fix when you can.\n\n<!-- hermes-ci-reminder head:f56231275cd10e79988ac1e26fd312a8c3abef0a -->

zamitto commented:
@Derval-Filho a PR ainda tem conflitos de merge, consegue atualizar sua PR com a branch main?

Derval-Filho commented:
eu nao sei como fazer essas correçoes


Em qui., 9 de abr. de 2026 às 14:50, Zamitto ***@***.***>
escreveu:

> *zamitto* left a comment (hydralauncher/hydra-themes#591)
> <https://github.com/hydralauncher/hydra-themes/pull/591#issuecomment-4216314414>
>
> @Derval-Filho <https://github.com/Derval-Filho> a PR ainda tem conflitos
> de merge, consegue atualizar sua PR com a branch main?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/hydralauncher/hydra-themes/pull/591#issuecomment-4216314414>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A3NSZ3VS6LXWTC7T67FS4OL4U7PFXAVCNFSM6AAAAACLD4A6SWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHM2DEMJWGMYTINBRGQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
ATT: Derval De Araujo Filho
***@***.***
***@***.***
61999017033

chubbygrannychaser commented:
@Derval-Filho your PR currently has failing CI checks (validate-themes). Could you take a look and push a fix? Head `f562312`.

<!-- hermes-ci-reminder head:f56231275cd10e79988ac1e26fd312a8c3abef0a -->

Derval-Filho commented:
> @Derval-FilhoSeu PR está apresentando falhas nas verificações de CI (validate-themes). Você poderia dar uma olhada e enviar uma correção? (Head `f562312`.)

Where can I find these f562312?

chubbygrannychaser reviewed (approved):
Theme submission looks good on \. Approving this revision.

chubbygrannychaser commented:
Correction from automation: the currently reported checks are green on head f562312, but this PR still is not mergeable under the repo moderation rules because mergeability is CONFLICTING and the required validate-themes check is not present/successful. <!-- hermes-moderation-note head:f56231275cd10e79988ac1e26fd312a8c3abef0a -->

Derval-Filho commented:
I don't know how to find the error you're pointing out; I don't know how to
fix it.

Em sáb., 2 de mai. de 2026 às 15:08, Chubby Granny Chaser <
***@***.***> escreveu:

> *chubbygrannychaser* left a comment (hydralauncher/hydra-themes#591)
> <https://github.com/hydralauncher/hydra-themes/pull/591#issuecomment-4364419409>
>
> Hi @Derval-Filho <https://github.com/Derval-Filho> — please take a look
> at the failing checks for head f562312
> <https://github.com/hydralauncher/hydra-themes/commit/f56231275cd10e79988ac1e26fd312a8c3abef0a>
> and push a fix when you can.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/hydralauncher/hydra-themes/pull/591#issuecomment-4364419409>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A3NSZ3WUELPNY2E742XPOK34YY2RZAVCNFSM6AAAAACLD4A6SWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHM2DGNRUGQYTSNBQHE>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
ATT: Derval De Araujo Filho
***@***.***
***@***.***
61999017033

chubbygrannychaser commented:
Hi @Derval-Filho — the required `validate-themes` check is currently missing on head `f562312`. Please update the PR so that check runs and passes before this can be merged.
